power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -103,8 +103,18 @@ class IDNotFound : public PowerGridError {
 
 class InvalidMeasuredObject : public PowerGridError {
   public:
-    InvalidMeasuredObject(const std::string& object, const std::string& sensor) {
-        append_msg(sensor + " is not supported for " + object);
+    InvalidMeasuredObject(std::string const& object, std::string const& sensor) {
+        append_msg(sensor + " measurement is not supported for object of type " + object);
```

Comment:
only if you don't modify them.

because `std::string_view` is an abstraction of any character sequence type and not an actual string, string operations like `operator+` are not defined. For those cases, `std::string` is still required

Reply:
If you put `{}`'s inside, that would be all of a sudden modern, wouldn't it? ;)

Reply:
yes but see this comment https://github.com/PowerGridModel/power-grid-model/issues/564#issuecomment-2039839342

> It is possible that MacOS needs an additional feature flag before we can use this feature: https://github.com/PowerGridModel/power-grid-model/actions/runs/8570641564/job/23489029489
> 
> Note that according to https://en.cppreference.com/w/cpp/compiler_support/20 , the XCode version we use in CI for MacOS (15.2) should support it, so this result was unexpected. See also https://github.com/PowerGridModel/power-grid-model/actions/runs/8570641564/job/23489028931#step:5:36

Reply:
i'll have a look at it when i have time